dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPIImpl.java

```
@@ -768,4 +801,80 @@ public void deleteContainerContentTypesByContainerInode(final Container containe
 	public void updateUserReferences(final String userId, final String replacementUserId)throws DotDataException, DotSecurityException{
 		containerFactory.updateUserReferences(userId, replacementUserId);
 	}
+
+	private  ResolvedPath resolvePath(final String inputPath,
+									  final User user,
+									  final boolean live,
+									  final boolean respectFrontEndEndRoles) throws DotSecurityException, DotDataException {
+
+
+		final FileAssetContainerUtil fileAssetContainerUtil = FileAssetContainerUtil.getInstance();
+		final String relativePath;
+
+		final Map<String, Host> hostsToFound = new LinkedHashMap<>();
+
+		if (fileAssetContainerUtil.isFullPath(inputPath)) {
+			final String hostName = fileAssetContainerUtil.getHostName(inputPath);
+			relativePath = fileAssetContainerUtil.getPathFromFullPath(hostName, inputPath);
+			final Host host = APILocator.getHostAPI().findByName(hostName, user, respectFrontEndEndRoles);
+
+			if (host != null) {
+				hostsToFound.put(host.getInode(), host);
+			}
+		} else {
+			relativePath = inputPath;
+		}
+
+		try {
+			final HttpServletRequest request = HttpServletRequestThreadLocal.INSTANCE.getRequest();
+
+			if (request != null) {
+				final Host currentHost = WebAPILocator.getHostWebAPI().getCurrentHost(request, user);
+				hostsToFound.put(currentHost.getInode(), currentHost);
+			}
+		} catch(DotSecurityException e) {

```

Codacy found an issue: [Avoid empty catch blocks](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)